tests/s3/test_object_tagging.py

```diff
@@ -1368,28 +1306,22 @@ def test_put_non_existing_object_tagging_2486(self):
     @CTFailOn(error_handler)
     def test_get_non_existing_object_tagging_2487(self):
         """Verify get object with tagging support to non-existing object."""
-        self.log.info(
-            "Verify get object with tagging support to non-existing object")
-        self.log.info("Creating a bucket with name %s",
-                      self.bucket_name)
-        resp = self.s3_test_obj.create_bucket(
-            self.bucket_name)
+        self.log.info("Verify get object with tagging support to non-existing object")
+        self.log.info("Creating a bucket with name %s", self.bucket_name)
+        resp = self.s3_test_obj.create_bucket(self.bucket_name)
         assert resp[0], resp[1]
         assert self.bucket_name == resp[1], resp[1]
-        self.log.info(
-            "Bucket is created with name %s",
-            self.bucket_name)
+        self.log.info("Bucket is created with name %s", self.bucket_name)
         self.log.info("Retrieving tags from non existing object")
         try:
-            self.tag_obj.get_object_with_tagging(
-                self.bucket_name,
-                self.object_name)
+            self.tag_obj.get_object_with_tagging(self.bucket_name, self.object_name)
         except CTException as error:
             assert S3_OBJ_TST["s3_object"]["key_err"] in str(
                 error.message), error.message
         self.log.info(
             "Verify get object with tagging support to non-existing object")
 
+    @pytest.mark.parallel
```

Comment:
Have you tested it by running tests in parallel? If not then don't use it for now.